source/upgrade.txt

```diff
@@ -66,12 +66,64 @@ and upgrade versions.
   If you're upgrading the {+driver-short+} from version 3.0 to version 3.10,
   address all breaking changes listed for versions 3.1 to 3.10, if any.
 
+Types of Breaking Changes
```

Comment:
Question/Suggestion: should we change the `.. contents::` component to be depth level 2? Each of the main headers have a few subsections so might be helpful to include those in the page ToC. If changing this results in the ToC becoming too lengthy/unwieldy, feel free to leave

Reply:
I changed depth to 2 originally, but it did make the TOC look overly complicated (since the subheaders have long titles). So I think I'll leave as is

Reply:
nvm changed back to 2 after restructuring!

Reply:
Possibly related (I do not fully understand the depth level discussion), I am in favor of removing the "Breaking Change Definition" and "Types of Breaking Changes" headers entirely while preserving the remaining content as-is, such that the current contents of each section just become contents of "Overview".

Reply:
Hm, Overview gets pretty long and disorganized if I remove those subheaders.

Reply:
If so, I am fine with leaving them as-is.